erddap/content/datasets.xml

```diff
@@ -2130,18 +2187,21 @@
       <addAttributes>
         <att name="ioos_category">Ice Distribution</att>
         <att name="long_name">Icebergs</att>
+        <att name="units">1</att>
       </addAttributes>
     </dataVariable>
     <dataVariable>
       <sourceName>SST</sourceName>
-      <destinationName>SST</destinationName>
+      <destinationName>sea_surface_temperature</destinationName>
       <dataType>float</dataType>
       <!-- sourceAttributes>
       </sourceAttributes -->
       <addAttributes>
         <att name="_FillValue" type="float">NaN</att>
         <att name="ioos_category">Temperature</att>
         <att name="long_name">Sea Surface Temperature</att>
+        <att name="standard_name">sea_surface_temperature</att>
+        <att name="units">1</att>
```

Comment:
The unit here should, likely, be degree_C

Reply:
I am happy to include units to all of these types of climate indexes where appropriate, but I want to be sure it represents what we think it represents.

When I looked at the included paper for this climate index, it wasn't readily apparent whether these are relative indexes from the climate average each year, i.e. if you would expect +5C median for sea surface temperature, then this would mean this value is anywhere from (less or more) -2.5C to + 2.5C i.e. indexed from that baseline temperature. Otherwise it could simply be the actual value, but looking at these ranges it might be by what is expected i.e. how come there is nothing larger than a 5 integer difference?

I wasn't certain so that's why I left it unitless because it could indicate more or less based on some other climate index parameter? Especially in the case of icebergs/sea ice cover, what does -2 or +1.3 icebergs mean? Two less than last year? Or is this a range of data referring to something else?

![image](https://user-images.githubusercontent.com/28519105/180510647-477f9631-8765-4628-9ca4-7b95050791ce.png)

Reply:
A more than fair point - I admit, I was paying more attention to the standard name and then inferring what unit "should" apply rather than examining what the data itself presented - which will be true of my other comments on this PR.

Rather than guessing amongst ourselves, we should contact the data provider to clarify how these values should be classified - it is possible there is no appropriate standard name for them currently.

It is more clear to me now why units 1 was selected, apologies.